### PR TITLE
added README.md

### PR DESCRIPTION
**Pull request by timweyel**

### CODE REVIEW

**Review by sydneywalcoff — COMMENTED** [review]
The description is rendering as a header. Do we want to make a specific description header and then the description below it?

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+3. [License](#license)
+4. [Contribution](#contribution)
+5. [Testing](#testing)
+6. [Questions](#questions)
+
+## Usage
+masterTicket is used by music enthusiasts to find the music events that interest them most.
+
+## License
+Covered by the MIT license.
```
Comment (squidbeaks) [inline README.md:20]: Is this true?

Comment (timweyel) [reply README.md:20]: not if you don't want to share it, i guess

Comment (squidbeaks) [reply README.md:20]: haha is it actually an MIT license?

Comment (timweyel) [reply README.md:20]: yeah, it's like the most 'lenient' license or whatever. https://en.wikipedia.org/wiki/MIT_License

**Pull request merged**